app/javascript/stylesheets/application.css

```
@@ -18,14 +18,18 @@
 
   h3,
   h4 {
-    @apply text-3xl md:text-4xl 2xl:text-5xl font-bold !important;
+    @apply text-3xl md:text-4xl 2xl:text-5xl font-bold !important mb-8;
```

swapping this just before the `!important`

```suggestion
    @apply text-3xl mb-8 md:text-4xl 2xl:text-5xl font-bold !important;
```